src/clj/harja/kyselyt/uusi_kustannussuunnitelma_kyselyt.sql

```diff
@@ -82,6 +108,16 @@ SELECT id,
 FROM kustannusarvioitu_tyo
 WHERE id = :id;
 
+-- name: hae-viimeisin-muokkaaja-erillishankinnoille
+SELECT greatest(kt.muokattu, kt.luotu) as viimeisin_muokkaus, concat(k.etunimi, ' ', k.sukunimi) as viimeisin_muokkaaja
```

Comment:
GREATEST vois olla isolla kuten muukin synktaksi
```suggestion
SELECT GREATEST(kt.muokattu, kt.luotu) as viimeisin_muokkaus, concat(k.etunimi, ' ', k.sukunimi) as viimeisin_muokkaaja
```

Reply:
Vaihdettu